src/marshmallow/utils.py

```diff
@@ -196,6 +231,25 @@ def to_iso_date(date: dt.date) -> str:
     return dt.date.isoformat(date)
 
 
+def to_iso_time(time: dt.time, format: str = "") -> str:
+    """Return the ISO8601-formatted representation of a time object.
+
+    :param time: The time.
+    :param format: An ISO8601 time format string.
+    """
+    if format:
+        format_exp = {
+            "hh": "%H",
```

Comment:
thought: Would it make sense for users to just pass the strftime format (`%H`, `%H:%M`, etc)?

Seems like that would keep consistency with the `format` argument of `fields.DateTime`.

Reply:
Absolutely. User should input strftime format. Let's not invent our own format.